specs-go/v1/manifest.go

```diff
@@ -36,3 +36,5 @@ type Manifest struct {
 	// Annotations contains arbitrary metadata for the image manifest.
 	Annotations map[string]string `json:"annotations,omitempty"`
 }
+
+const ScratchConfigDigestSHA256 = "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
```

Comment:
Is it worth also including something like `ScratchConfigData = "{}"` to make this abundantly clear / give implementers a constant for if they want/need to fall back to pushing this blob?

Reply:
Alternatively, this could instead be something like `ScratchConfigDescriptor` and be a full content descriptor with Digest, Size, and the optional Data field, maybe with a null or `application/json` media type?

Reply:
how about that comment?